cli/readme.py

```diff
@@ -485,7 +485,12 @@ def write_job_workflow(job):
         workflow_yaml += f"""          bash -x generate-yml.sh\n"""
         # workflow_yaml += f"""          bash -x {os.path.relpath(".", project_dir)}/run-job.sh generate-yml.yml\n"""
     workflow_yaml += f"""          bash -x {os.path.relpath(".", project_dir).replace(os.sep, "/")}/run-job.sh {filename}.yml
-      working-directory: cli/{posix_project_dir}\n"""
+      working-directory: cli/{posix_project_dir}
```

Comment:
Do we want this in `sdk/python/readme.py`, and `tutorials/readme.py` too?

Reply:
yes, I'm adding it there, but I wanted to make sure I got it working in at least one first.